tensorflow/lite/experimental/micro/simple_tensor_allocator.cc

```diff
@@ -135,10 +135,13 @@ TfLiteStatus SimpleTensorAllocator::AllocateTensor(
       src_quantization->zero_point() &&
       (src_quantization->zero_point()->size() > 0)) {
     result->params.scale = src_quantization->scale()->Get(0);
-    result->params.zero_point = src_quantization->zero_point()->Get(0);
+    memcpy(&result->params.zero_point,
```

Comment:
We're trying to avoid memcpy where possible, since on some toolchains it requires libc linker support. Could you replace it with either an explicit loop or a small utility function? Thanks!

Reply:
I've changed this to a loop, to remove any library dependencies.